include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1083,9 +1083,9 @@ int bt_audio_codec_cfg_meta_get_val(const struct bt_audio_codec_cfg *codec_cfg,
  * @param data       Pointer to the data-pointer to set.
  * @param data_len   Length of @p data.
  *
- * @retval The meta_len of @p codec_cfg on success
- * @retval -EINVAL if arguments are invalid
- * @retval -ENOMEM if the new value could not set or added due to memory
+ * @retval meta_len The meta_len of @p codec_cfg on success
```

Comment:
Shall we do the same as for the data_len field of codec_cfg?

```suggestion
 * @retval meta_len The @p codec_cfg.meta_len on success
```

Reply:
Good catch, will update